contrib/python/src/python/pants/contrib/python/checks/checker/class_factoring.py

```diff
@@ -13,7 +13,7 @@ class ClassFactoring(CheckstylePlugin):
   """Enforces recommendations for accessing class attributes.
 
   Within classes, if you see:
-    class Distiller(object):
+    class Distiller:
```

Comment:
There are a couple more that need changes still. Anything with `contrib/python/checks/checker` in the path. I hadn't flagged them each because there are more than a few.